core/src/test/scala/unit/kafka/utils/JsonTest.scala

```diff
@@ -85,8 +73,8 @@ class JsonTest {
     assertEquals("{}", Json.encode(Map()))
     assertEquals("{\"a\":1,\"b\":2}", Json.encode(Map("a" -> 1, "b" -> 2)))
     assertEquals("{\"a\":[1,2],\"c\":[3,4]}", Json.encode(Map("a" -> Seq(1,2), "c" -> Seq(3,4))))
-    assertEquals("\"str1\\\\,str2\"", Json.encode("str1\\,str2"))
-    assertEquals("\"\\\"quoted\\\"\"", Json.encode("\"quoted\""))
+    assertEquals(""""str1\\,str2"""", Json.encode("""str1\,str2"""))
+    assertEquals(""""\"quoted\""""", Json.encode(""""quoted""""))
```

Comment:
Ouch, maybe my suggestion was bad in this line.

Reply:
A bit unclear to be honest. Both options seem hard to read. :)

Reply:
Shall I leave it as is or do you want me to change it back - I don't mind either way.